tests/test_format_notebook_content.py

```diff
@@ -20,6 +21,43 @@ def test_format_notebook_content():
     )
 
 
+def test_format_single_notebook():
+    """
+    Test formatting a single notebook in an directory of unformatted notebooks
+    formats only that notebook
+    """
+    data_path = pathlib.Path(f"{__file__}").parent / "data"
+    source_notebook_path = data_path / "unformatted" / "spaces.ipynb"
+    copy_notebook_path = data_path / "unformatted" / "spaces_copy.ipynb"
+    shutil.copy(str(source_notebook_path), str(copy_notebook_path))
+
+    source_json = json.loads(source_notebook_path.read_text())
+    copy_json = json.loads(copy_notebook_path.read_text())
+
+    assert all(
+        [
+            copy_cell["source"] == source_cell["source"]
+            for copy_cell, source_cell in zip(copy_json["cells"], source_json["cells"])
+        ]
+    )
+
+    # Have to run blackbook main to show spaces_copy.ipynb is unaffected
+    blackbook.__main__.main(source_notebook_path)
+    output_json = json.loads(source_notebook_path.read_text())
+    check_copy_json = json.loads(copy_notebook_path.read_text())
```

Comment:
do you need to create `check_copy_json`? Why not just used `copy_json` that you defined above?

Reply:
If

```python
blackbook.__main__.main(source_notebook_path)
```

failed to act on only the source notebook it is being run on, then the default behavior of `blackbook` would make it format _all_ notebooks in the directory. This is proof that the test isn't using information _before_ the formatting took place.

Reply:
That's fantastic. Thank you for clarifying 😄

Reply:
Sure! :) Happy to.